aio/content/marketing/contributors.json

```
@@ -114,7 +114,7 @@
     "picture": "antonio-cardenas.jpg",
     "twitter": "yeoudev",
     "website": "https://yeou.dev",
-    "bio": "Antonio is a senior Angular developer from Honduras, focus on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Love to share knowledge Typescript Coffe and Cats 🐱.",
+    "bio": "Antonio is a senior Angular developer from Honduras, focus on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Love to share knowledge Typescript Coffee and Cats 🐱.",
```

I just noticed a bit weird grammar it this sentence. I think it should be more like this:
```suggestion
    "bio": "Antonio is a senior Angular developer from Honduras, who focuses on immersive experiences and Web Performance, He is a public Speaker, Mentor and Facilitator for entrepreneurship programs as Techstars Startup Weekend, Content Creator and tech writer Cofounder of non profit organization @Macaotech, Loves to share knowledge Typescript Coffee and Cats 🐱.",
```

What do you think?

---

Yes, it can be improved.

---

https://github.com/angular/angular/pull/51527/commits/7f4127e36512086bdd827c696de3c3fbfac62573